packages/reusable-blocks/src/private-apis.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * WordPress dependencies
+ */
+import { __dangerousOptInToUnstableAPIsOnlyForCoreModules } from '@wordpress/private-apis';
+
+export const { lock, unlock } =
```

Comment:
Technically I don't think `lock` is being used in this package?

Reply:
removed